sklearn/feature_selection/_rfe.py

```diff
@@ -566,5 +585,23 @@ def fit(self, X, y, groups=None):
 
         # Fixing a normalization error, n is equal to get_n_splits(X, y) - 1
         # here, the scores are normalized by get_n_splits(X, y)
-        self.grid_scores_ = scores[::-1] / cv.get_n_splits(X, y, groups)
+        grid_scores = scores[::-1] / cv.get_n_splits(X, y, groups)
+        self.cv_results_ = {}
+        for i in range(grid_scores.shape[0]):
+            key = "split{}_score".format(i)
+            self.cv_results_[key] = grid_scores[i]
+        self.cv_results_["mean_score"] = np.mean(grid_scores, axis=0)
+        self.cv_results_["std_score"] = np.std(grid_scores, axis=0)
         return self
+
+    @deprecated(
+        "The grid_scores_ attribute is deprecated in version 0.23 in favor "
+        "of cv_results_ and will be removed in version 0.25"
+    )
+    @property
+    def grid_scores_(self):
+        # remove 2 for mean_score, std_score
+        grid_size = len(self.cv_results_) - 2
+        return np.asarray(
+            [self.cv_results_["split{}_score".format(i)]
+            for i in range(grid_size)]).T
```

Comment:
i think you wrong indented... and newline missing.

sklearn/feature_selection/_rfe.py:607:13: E128 continuation line under-indented for visual indent
            for i in range(grid_size)]).T            ^
sklearn/feature_selection/_rfe.py:607:42: W292 no newline at end of file
            for i in range(grid_size)]).T                                         ^

Exited with code exit status 1